android/quest/src/test/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModelTest.kt

```diff
@@ -1448,4 +1448,72 @@ class QuestionnaireViewModelTest : RobolectricTest() {
 
     Assert.assertTrue(questionnaireResponse.item.size <= questionnaire.item.size)
   }
+  @Test
+  fun testLoadQuestionnaireShouldUReturnCorrectItemsWithUpdateOnEdit() {
```

Comment:
How are we testing that ActionParameterType.UPDATE_DATE_ON_EDIT were indeed returned? I'm only seeing the verification that `loadQuestionnaire` is called.

Reply:
I have enhanced the test